index.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+  <title>My Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <section class="main-header">
+    
+      <nav class="toolbar-container">
+        
+        <a class="toolbar-logo">
+          <p class="Logo">My Logo</p>
+        </a>
+
+        <a class="toolbar-button">
+          <img src="images/header/toolbar/Union-button.png" alt="toolbar button">
+        </a>
+      </nav>
+    
```

Comment:
- [ ] For all your `a` tags, you should add `href=""` attribute, if you don't have an available link to put inside your link tag `a` as href value, you can use "#" as href value. For instance:
```
<a href=#>
  <img ....>
</a>
```

Reply:
- This has been resolved.